backend/src/user/user.controller.ts

```diff
@@ -0,0 +1,21 @@
+import { Controller, Get, Param } from '@nestjs/common';
+import { UserService } from './user.service';
+
+@Controller('/users')
+export class UserController {
+constructor(private readonly userService: UserService) {
+}
+    @Get(':id')
+    getUserById(@Param('id') id: string)
+    {
+        const data = {id: "id"}
+        return id;
+    }
+
+    @Get('/byMail:mail')
```

Comment:
to będzie działać ale nazwy edpointów raczej samymi małymi literami się pisze i jeszcze jest kwestia tego że przy tym podejściu email jest dopisywany bezpośrednio do adresów. Tutaj bardziej rozsądne wydaje się użycie parametrów (które de facto też są dopisywane do adresu, ale jednak w sekcji parametrów a nie w 1 członie):
![image](https://user-images.githubusercontent.com/37194246/161113475-3b050317-1b67-4cbb-a830-56229252d537.png)